CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Qobuz App Change Log
 
 * Quobuz is unsupported until a new maintainer is found. Issue: tiliado/nuvolaruntime#634
 
+1.3 - October 4th, 2020
+------------------------
```

Comment:
We write the release number and date when a release is made. I've changed it to `unreleased`.